lte/gateway/python/integ_tests/s1aptests/util/traffic_util.py

```diff
@@ -409,7 +409,7 @@ def _iface_up_ipv6(ip):
             ifname=TrafficTest._net_iface_ipv6,
         )[0]
         TrafficTest._iproute.addr(
-            'add', index=net_iface_index, address=ip.exploded,
+            'add', index=net_iface_index, address=ip.exploded, prefixlen=128
```

Comment:
FYI the formatter is complaining that there should be a comma at the end of this line.